extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -38,12 +47,12 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
    * This configuration controls whether to verify the privilege objects in single call or
    * to verify them one by one.
    */
-  def authorizeInSingleCall: Boolean = getRangerConf.getBoolean(
-    s"ranger.plugin.${getServiceType}.authorize.in.single.call",
+  def authorizeInSingleCall: Boolean = getRangerConf(defaultBasePlugin).getBoolean(
+    s"ranger.plugin.${defaultBasePlugin.getServiceType}.authorize.in.single.call",
```

Comment:
why use the defaultBasePlugin.getServiceType?

Reply:
Compatible to exsiting config name as `ranger.plugin.spark.authorize.in.single.call`.
Or let me use `serviceType` with `spark` value instead.